datalad/interface/diff.py

```diff
@@ -115,7 +116,8 @@ def _get_untracked_content(dspath, report_untracked, paths=None):
             # strip state marker
             line[3:])
         norm_apath = normpath(apath)
-        if paths and not any([norm_apath == p or apath.startswith(_with_sep(p)) for p in paths]):
+        if paths and not any(norm_apath == p or path_startswith(apath, p)
+                             for p in paths):
```

Comment:
Same here, I think.

Reply:
Actually not true. that normpath confused me. Here it is okay, since equality is tested anyway.
The question remains though, why to use normpath for equality testing, but not for startswith? That doesn't make sense to me.
Smells like a misunderstanding of what normpath does.